src/main/java/hudson/plugins/analysis/core/AbstractResultAction.java

```diff
@@ -104,12 +106,12 @@ public ToolTipProvider getToolTipProvider() {
      *
      * @return the associated build of this action
      */
-    public final AbstractBuild<?, ?> getOwner() {
+    public final Run<?, ?> getOwner() {
```

Comment:
You'll find a lot of changes like this one. This method needs to return ```Run``` to properly work inside a Workflow job. So what options do we have?